conf/command.conf

```diff
@@ -10,57 +10,49 @@ CustomCommand {
                          # Currently support only 1 key with first argument.
         minArgs 3	 # minimum arguments(including command itself), default is 2(key only)
         maxArgs 3        # maximum arguments(including command itself), default is 2(key only)
-        mode Read,Write  # behavior or permission. Currently support Read, Write, Admin
+        mode Read,Write  # a command mode Read/Write/Admin is exclusive, Conf.cpp should check this.
```

Comment:
I means Read/Write/Admin can't appear at the same mode.
Conf.cpp check this means we should code to check this, not user to check this.

Reply:
I see, it was applied. I learned github commit attitudes from your review. 😅